### PR TITLE
fix: #546 issue with closing registration

### PR DESCRIPTION
**Pull request by danorlando**
Fixes issue #546 where closing registration was no longer working.
Also renames some controller files for consistency.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Set ALLOW_REGISTRATION to false and verify that registration is not possible. Then set it to true and make sure it works. Also manually entered /register path in url bar to make sure that it reroutes to /login when registration is disabled.

### CODE REVIEW

**Review by fuegovic — APPROVED** [review]
I tested and can confirm that the fix is working

**Comment by bsu3338** [comment]
I just saw openid merged. It will need the same fix in the Login to be consistent with google auth in this PR
 === true

File: client/src/components/Auth/Registration.tsx
Comment (danny-avila) [inline client/src/components/Auth/Registration.tsx:1]: As @bsu3338 mentioned, the openId auth handling needs to be added and then this all set to go!

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: client/src/components/Auth/Registration.tsx
Comment (danorlando) [reply client/src/components/Auth/Registration.tsx:1]: This can be merged now.

**Review by danny-avila — APPROVED** [review]

**Pull request merged**